### ISSUE TITLE
Missing Approve and Reject buttons in review

### ISSUE THREAD

**Issue by endorama** [sindresorhus/refined-github#1646]
Hello, thank you for this project. 

Today while trying to review a PR I struggled finding the `Approve` and `Reject` buttons: they were missing from the review popup completely. I then tried to disable the `add-quick-review-buttons` feature, and discovered that was due to them being `disabled` (I was the PR opener and we have some protection in place to avoid PR owners merging their PRs without approval).

This behaviour is intended, as demonstrated by line
https://github.com/sindresorhus/refined-github/blob/6f8490f76384d84f146436775eb58067896abbee/source/features/add-quick-review-buttons.js#L31-L32
which exclude radios in disabled state.

To have a nicer user experience in such cases I think would be helpful to move disabled radios to buttons in disabled state, using the provided help text GitHub adds to the radios.

The HTML structure for the disabled radios has a `aria-label` containing the popup text
```html
<div class="form-checkbox mx-3 my-2">
    <label class="d-block text-gray-light tooltipped tooltipped-ne tooltipped-no-delay tooltipped-left" 
           aria-label="Pull request authors can’t approve their own pull request">
    <input type="radio" name="pull_request_review[event]" value="approve" disabled="">
    Approve
    <p class="text-normal text-gray-light">Submit feedback and approve merging these changes.</p>
    </label>
</div>
```

I can provide a PR to enhance this behaviour is you are ok with that.

Thanks a lot!

**Comment by bfred-it** [comment]
Not a bad idea.

For reference, this is what it looks like normally

<img width="426" alt="screenshot 2018-11-30 at 23 09 01" src="https://user-images.githubusercontent.com/1402241/49297118-f4240900-f4f4-11e8-8163-e9309ea666d1.png">

**Comment by endorama** [comment]
That's great! I don't think I'll be able to work on it until mid December,
but if will be open still I'll code the feature.

Thanks!

**Comment by Guy-Del-Mistro** [comment]
Same issue. The Approve (etc.) is hard to find and I had to Google the second time again to make sure I was doing this right.
The PR was reused but when I went to add a 'dummy' review comment, just for sake of Approval (knowing the history of the fixes made), there were no actions provided.
Why is there not an obvious Approve/Reject option on the main PR comments page, as there was when I worked with Atlassian/Stash previously?
